docs/Operators.md

```diff
@@ -11663,43 +11663,161 @@ for test_name, shape in test_cases.items():
 
 #### Version
 
-This version of the operator has been available since version 10 of the default ONNX operator set.
+This version of the operator has been available since version 11 of the default ONNX operator set.
+
+Other versions of this operator: <a href="Changelog.md#Resize-10">Resize-10</a>
 
 #### Attributes
 
 <dl>
+<dt><tt>align_corners</tt> : int (default is 0)</dt>
+<dd>If set to 0 (default), the input and output tensors are aligned by the center points of their corner pixels. If set to 1, the input and output tensors are aligned by the corner points of their corner pixels, and the interpolation uses edge value padding for out-of-boundary values. For each dimension, denote x_resized as the coordinate in the resized tensor, x_original as the corresponding coordinate in the original tensor, length_original as the length of the original tensor in the specific dimension, length_resized as the length of the resized tensor in the specific dimension, if align_corners is 0, x_original = (x_resized + 0.5) / scale - 0.5, if align_corners is 1, x_original = x_resized * (length_original - 1) / (length_resized - 1).</dd>
+<dt><tt>cubic_coeff_a</tt> : float (default is -0.75)</dt>
+<dd>The coefficient 'a' used in cubic interpolation. Two common choice are -0.5 (in TensorFlow) and -0.75 (in PyTorch). Check out https://ieeexplore.ieee.org/document/1163711 for the details.</dd>
+<dt><tt>exclude_outside</tt> : int (default is 0)</dt>
+<dd>If set to 1, the weight of sampling locations outside the tensor will be set to 0 and the weight will be renormalized so that their sum is 1.0. The default value is 0.</dd>
 <dt><tt>mode</tt> : string (default is nearest)</dt>
-<dd>Two interpolation modes: nearest (default), and linear (including bilinear, trilinear, etc)</dd>
+<dd>Three interpolation modes: nearest (default), linear and cubic. The "linear" mode includes linear interpolation for 1D tensor and N-D linear interpolation for N-D tensor (For example, trilinear interpolation for 2D tensor). The "cubic" mode includes cubic interpolation for 1D tensor and N-D cubic interpolation for N-D tensor (For example, tricubic interpolation for 2D tensor).</dd>
 </dl>
 
-#### Inputs
+#### Inputs (2 - 3)
 
 <dl>
-<dt><tt>X</tt> : T</dt>
+<dt><tt>X</tt> : T1</dt>
 <dd>N-D tensor</dd>
-<dt><tt>scales</tt> : tensor(float)</dt>
-<dd>The scale array along each dimension. It takes value greater than 0. If it's less than 1, it's sampling down, otherwise, it's upsampling. The number of elements of 'scales' should be the same as the rank of input 'X'.</dd>
+<dt><tt>scales</tt> : T2</dt>
```

Comment:
roi missing?

Reply:
Thanks! I have not re-generate the doc yet.

Reply:
Done. Thanks!